projects/plugins/social/src/js/components/header/index.js

```diff
@@ -12,38 +12,38 @@ import {
 	ShareLimitsBar,
 	store as socialStore,
 	useShareLimits,
+	getSocialScriptData,
 } from '@automattic/jetpack-publicize-components';
-import { getScriptData } from '@automattic/jetpack-script-data';
+import { getAdminUrl } from '@automattic/jetpack-script-data';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import { Icon, postList } from '@wordpress/icons';
 import StatCards from '../stat-cards';
 import styles from './styles.module.scss';
 
 const Header = () => {
-	const connectionData = window.jetpackSocialInitialState.connectionData ?? {};
 	const {
-		connectionsAdminUrl,
+		// TODO replace some of these from script data (initial state)
 		hasConnections,
 		isModuleEnabled,
-		newPostUrl,
 		postsCount,
 		totalShareCount,
 		showShareLimits,
 	} = useSelect( select => {
 		const store = select( socialStore );
 		return {
-			connectionsAdminUrl: connectionData.adminUrl,
 			hasConnections: store.getConnections().length > 0,
 			isModuleEnabled: store.isModuleEnabled(),
-			newPostUrl: `${ store.getAdminUrl() }post-new.php`,
 			postsCount: store.getSharedPostsCount(),
 			totalShareCount: store.getTotalSharesCount(),
 			showShareLimits: store.showShareLimits(),
 		};
 	} );
-	// TODO - Replace this with a utility function like `getSocialFeatureFlags` when available
-	const { useAdminUiV1 } = getScriptData().social.feature_flags;
+
+	const {
+		feature_flags: { useAdminUiV1 },
+		urls,
```

Comment:
Should we do destructing for both or neither?

Reply:
I left `urls` like that to maintain the context when the variable is used, i.e. `urls.connectionsManagementPage` conveys more information than just `connectionsManagementPage`